<?xml version="1.0"?>
<launch>
    <env name="GMON_OUT_PREFIX" value="ompl_planner" />
    <node pkg="mav_planning" type="global_planner_node" name="mav_planning" output="screen">
        <rosparam file="$(find mav_planning)/cfg/config.yaml" command="load" ns="" />
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find mav_planning)/rviz/config.rviz" />

</launch>